infra/blueprint-test/pkg/krmt/krm.go

```diff
@@ -58,6 +59,12 @@ func WithDir(dir string) krmtOption {
 	}
 }
 
+func WithResourcesDir(rscsDir string) krmtOption {
```

Comment:
Should we do `WithAdditionalResources(res ...string)` instead? `WithAdditionalResources` name implies these are additional resources and  `res` can be a file or dir and multiple res can be passed in.

Reply:
Makes sense.